### PR TITLE
EOS-18817, EOS-19760: Fix Firewall Issues for 28100 port

### PR DESCRIPTION
**Pull request by rahuljyoti65**
Signed-off-by: Rahul Jyoti <rahul.jyoti@seagate.com>

-----
[View rendered docs/User-Guides/VM-Deployment.md](https://github.com/rahuljyoti65/cortx-prvsnr/blob/EOS-18817/docs/User-Guides/VM-Deployment.md)

### CODE REVIEW

File: cli/src/virtual_appliance/bootstrap.sh [hunk 1]
```diff
@@ -31,6 +31,33 @@ trap trap_handler ERR
...
+#reconfigure lustre
+salt-call state.apply components.misc_pkgs.lustre.stop
+salt-call state.apply components.misc_pkgs.lustre.config
+salt-call state.apply components.misc_pkgs.lustre.start
+
+#reconfigure hare CDF
+salt-call state.apply components.hare.prepare
+
+#reconfigure CSM ha params (Not necessary on single node)
+# salt-call state.apply components.csm.prepare
```
Comment (ypise) [inline cli/src/virtual_appliance/bootstrap.sh:59]: This is not required in R2, so delete these lines.

File: cli/src/virtual_appliance/bootstrap.sh [hunk 2]
```diff
@@ -31,6 +31,33 @@ trap trap_handler ERR
...
+salt-call state.apply components.system.firewall.teardown
+salt-call state.apply components.system.firewall
+
+#reconfigure lustre
+salt-call state.apply components.misc_pkgs.lustre.stop
+salt-call state.apply components.misc_pkgs.lustre.config
+salt-call state.apply components.misc_pkgs.lustre.start
+
+#reconfigure hare CDF
+salt-call state.apply components.hare.prepare
```
Comment (ypise) [inline cli/src/virtual_appliance/bootstrap.sh:56]: This is not required in R2, so delete these lines.
Instead add 
```
salt-call state.apply components.hare.config
```

**Review by ypise — CHANGES_REQUESTED** [review]

File: cli/src/virtual_appliance/bootstrap.sh (hunk 1 above)
Comment (rahuljyoti65) [reply cli/src/virtual_appliance/bootstrap.sh]: Done

File: cli/src/virtual_appliance/bootstrap.sh (hunk 2 above)
Comment (rahuljyoti65) [reply cli/src/virtual_appliance/bootstrap.sh]: Done

**Review by ypise — APPROVED** [review]

**Review by ypise — APPROVED** [review]

**Comment by ypise** [comment]
Something has gone wrong with these PR.
I feel there are unrelated changes here.
@rahuljyoti65 do recheck.

**Comment by rahuljyoti65** [comment]
> Something has gone wrong with these PR.
> I feel there are unrelated changes here.
> @rahuljyoti65 do recheck.

@ypise I have rebased the PR on pre-cortx-1.0 branch. Now it should be good to merge.

**Review by ypise — APPROVED** [review]

File: cli/src/virtual_appliance/bootstrap.sh [hunk 3]
```diff
@@ -20,7 +20,7 @@
 set -euE
 
 export LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/va_bootstrap.log}"
-mkdir -p $(dirname ""${LOG_FILE}")"
+mkdir -p $(dirname "${LOG_FILE}")"
```
Comment (ypise) [inline cli/src/virtual_appliance/bootstrap.sh:23]: The trailing `"`?
That needs to be removed as well.

**Review by ypise — CHANGES_REQUESTED** [review]

File: cli/src/virtual_appliance/bootstrap.sh (hunk 3 above)
Comment (rahuljyoti65) [reply cli/src/virtual_appliance/bootstrap.sh]: Oh ! I had missed that.
Done.

**Review by ypise — APPROVED** [review]

**Pull request merged**